### ISSUE TITLE
1.17: WorldServer.getChunkAt(int, int) not properly remapped from Spigot

### ISSUE THREAD

**Issue by Joo200** [PaperMC/Paper#5855]
### Expected behavior

Working WorldEdit, no stacktrace

### Observed/Actual behavior

```
[22:25:46 INFO]: Joo200 issued server command: //set beehive[honey_level=5]
[22:25:46 ERROR]: An unexpected error while handling a WorldEdit command
java.lang.NoSuchMethodError: 'net.minecraft.world.level.chunk.Chunk net.minecraft.server.level.WorldServer.getChunkAt(int, int)'
        at com.sk89q.worldedit.bukkit.adapter.impl.Spigot_v1_17_R1.getBlock(Spigot_v1_17_R1.java:349) ~[?:?]
        at com.sk89q.worldedit.bukkit.BukkitWorld.getFullBlock(BukkitWorld.java:484) ~[?:?]
```
Looks like WorldServer.getChunkAt isn't included at all, I'm unable to find it in my paper setup. I used the linked experimental build from the discord (#!1-17)

### Steps/models to reproduce

- Use the experimental 1.17 dev build
- Install WorldEdit with 1.17 compatibility: https://builds.enginehub.org/job/worldedit?branch=version/7.2.x
- Select an area and use //set stone

### Plugin list

WorldEdit
WorldGuard (not needed)

### Paper version

[22:40:47 INFO]: This server is running Paper version git-Paper-"3740db0" (MC: 1.17) (Implementing API version 1.17-R0.1-SNAPSHOT)
Error obtaining version information

### Agreements

- [X] I am running the latest version of Paper available from https://papermc.io/downloads.
- [X] I have searched for and ensured there isn't already an open issue regarding this.
- [X] My version of Minecraft is supported by Paper.

**Comment by Aurelien30000** [comment]
Confirmed, it works as expected on Spigot but not on Paper.

**Comment by Proximyst** [comment]
Assigning to @DenWav. I believe you are already aware of this.

**Comment by Joo200** [comment]
Looks like it's fixed on 8236add8cbecad5d7a5d87c1db1038523c39da7a (or earlier)

I compiled the experimental jar on my own and can now use WorldEdit as expected. Leaving open for DenWav to confirm this bug is really fixed

**Comment by DenWav** [comment]
Relevant issue: PaperMC/paperweight#17
Fixed in https://github.com/PaperMC/paperweight/commit/7013e8c2201a7d84f9b85276e4f6d2fd2d2f6229
